templates/grants/list.html

```diff
@@ -0,0 +1,99 @@
+{% extends "base.html" %}
+{% import "macros/macros.html" as macros %}
+
+{% block title %}
+  <meta name="og:title" content="Grants - {{ config.title }}">
+  <title>Grants - {{ config.title }}</title>
+{% endblock title %}
+
+{% block description %}
+  <meta name="og:description" content="Grants facilitated by the Urbit Foundation.">
+  <meta name="description" content="Grants facilitated by the Urbit Foundation.">
+{% endblock description %}
+
+{% block content %}
+{% set posts = get_section(path="community/grants-and-bounties/_index.md") %}
+<!-- header -->
+<nav class="mt4 full fourth-1-lg mb3 mb0-m mb0-l mb0-xl fixed-xl">
+  <a class="gray3" href="/">Urbit</a>
+  <span class="gray3"> / </span>
+  <a href="/community/grants-and-bounties/">Grants</a>
+
+  {% include "partials/navigation-search.html" %}
+
+</nav>
+<section class="full c4-10-lg pt4 lh-copy">
+
+  <h1>Urbit Grants Program</h1>
+
+  <p>Urbit is a community project. While anyone can contribute, we help focus development and reward exceptional contribution through our grants and bounties program.</p>
```

Comment:
```suggestion
  <p>Urbit is a community project. While anyone can contribute, we help focus development and reward exceptional contribution through our grants program.</p>
```
A bounty is a type of grant.